aiida/cmdline/commands/group.py

```diff
@@ -29,556 +34,281 @@ def __init__(self):
         list.
         """
         self.valid_subcommands = {
-            'list': (self.group_list, self.complete_none),
-            'show': (self.group_show, self.complete_none),
-            'description': (self.group_description, self.complete_none),
-            'create': (self.group_create, self.complete_none),
-            'rename': (self.group_rename, self.complete_none),
-            'delete': (self.group_delete, self.complete_none),
-            'addnodes': (self.group_addnodes, self.complete_none),
-            'removenodes': (self.group_removenodes, self.complete_none),
+            'list': (self.cli, self.complete_none),
+            'show': (self.cli, self.complete_none),
+            'description': (self.cli, self.complete_none),
+            'create': (self.cli, self.complete_none),
+            'rename': (self.cli, self.complete_none),
+            'delete': (self.cli, self.complete_none),
+            'addnodes': (self.cli, self.complete_none),
+            'removenodes': (self.cli, self.complete_none),
         }
 
-    def group_create(self, *args):
-        """
-        Create a new empty group.
-        """
-        if not is_dbenv_loaded():
-            load_dbenv()
+    @staticmethod
+    def cli(*args):  # pylint: disable=unused-argument
+        verdi.main()
 
-        import argparse
-        from aiida.orm import Group as G
 
-        parser = argparse.ArgumentParser(
-            prog=self.get_full_command_name(),
-            description='Create a new empty group.')
-        parser.add_argument('GROUPNAME', help="The name of the new group")
-
-        args = list(args)
-        parsed_args = parser.parse_args(args)
+@verdi_group.command("removenodes")
+@options.GROUP()
+@arguments.NODES()
+@with_dbenv()
+def group_removenodes(group, nodes, *args):
+    """
+    Remove NODES from a given AiiDA group.
+    """
+    from aiida.cmdline import wait_for_confirmation
 
-        group_name = parsed_args.GROUPNAME
+    group_pk = group.pk
+    group_name = group.name
```

Comment:
These variables are only used once and they are not shorter than using `group.pk` directly, so maybe just dereference directly?